Documentation/doc/Documentation/windows.txt

```diff
@@ -3,7 +3,7 @@
 \cgalAutoToc
 
 \cgal \cgalReleaseNumber is supported for the following \ms Visual `C++` compilers:
-14.0, 15.9, 16.0 (\visualstudio 2015, 2017, and 2019).
+14.0, 15.9, 16.0, 19.0 (\visualstudio 2015, 2017, 2019, and 2022).
```

Comment:
So far, the versions we document are Visual Studio versions. Maybe we should document MSVC versions instead, but then we would have to change that for all versions.
```suggestion
14.0, 15.9, 16.0, 17.0 (\visualstudio 2015, 2017, 2019, and 2022).
```

https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-notes